docs/api/en/core/BufferGeometry.html

```diff
@@ -222,7 +222,9 @@ <h3>[method:BufferAttribute deleteAttribute]( [param:String name] )</h3>
 		<h3>[method:undefined dispose]()</h3>
 		<p>
 		Disposes the object from memory. <br />
-		You need to call this when you want the BufferGeometry removed while the application is running.
+		When the geometry is rendered it allocates WebGL resources which are not implicitly freed.
+		You need to call this when you want the BufferGeometry removed while the application is running. <br />
+		For more details read: [link:https://threejs.org/docs/index.html#manual/en/introduction/How-to-dispose-of-objects How to dispose of objects]
```

Comment:
I would rephrase this section and write it more compact:

> When the geometry is rendered, it allocates WebGL resources which can only be disposed of by calling this method. For more details read: https://threejs.org/docs/index.html#manual/en/introduction/How-to-dispose-of-objects

Reply:
Agreed.